From 5c5e5e83918b23709d1135299596b968feff00bc Mon Sep 17 00:00:00 2001
From: Zhigang Gong <zhigang.gong@linux.intel.com>
Date: Tue, 2 Jul 2013 18:49:49 +0800
Subject: [PATCH 04/10] utests: increase local size in the two barrier test
 cases.
To: beignet@lists.freedesktop.org

Increasing the local size to 256 to bring more pressure
to barrier testing.

Signed-off-by: Zhigang Gong <zhigang.gong@linux.intel.com>
---
 utests/compiler_global_memory_barrier.cpp  |    2 +-
 utests/compiler_local_memory_barrier_2.cpp |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/utests/compiler_global_memory_barrier.cpp b/utests/compiler_global_memory_barrier.cpp
index a6496a7..ea84e72 100644
--- a/utests/compiler_global_memory_barrier.cpp
+++ b/utests/compiler_global_memory_barrier.cpp
@@ -13,7 +13,7 @@ static void compiler_global_memory_barrier(void)
 
   // Run the kernel
   globals[0] = n/2;
-  locals[0] = 32;
+  locals[0] = 256;
   OCL_NDRANGE(1);
   OCL_MAP_BUFFER(0);
 
diff --git a/utests/compiler_local_memory_barrier_2.cpp b/utests/compiler_local_memory_barrier_2.cpp
index b074123..4fa090b 100644
--- a/utests/compiler_local_memory_barrier_2.cpp
+++ b/utests/compiler_local_memory_barrier_2.cpp
@@ -5,7 +5,7 @@ static void compiler_local_memory_barrier_2(void)
   const size_t n = 16*1024;
 
   globals[0] = n/2;
-  locals[0] = 32;
+  locals[0] = 256;
 
   // Setup kernel and buffers
   OCL_CREATE_KERNEL("compiler_local_memory_barrier_2");
-- 
1.7.10.4

